authmate/session_tokens.go

```diff
@@ -4,15 +4,14 @@ import (
 	"encoding/json"
 	"fmt"
 
-	apisession "github.com/nspcc-dev/neofs-api-go/v2/session"
 	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
 	"github.com/nspcc-dev/neofs-sdk-go/session"
 )
 
 type (
 	sessionTokenModel struct {
 		Verb        string `json:"verb"`
-		ContainerID string `json:"ContainerID"`
+		ContainerID string `json:"containerID"`
```

Comment:
Is it safe to change then name?

Reply:
yes, explained in the commit

Reply:
Sorry, I watched the whole PR via `Files changed`